orbit/cmd/orbit/orbit.go

```diff
@@ -891,7 +892,7 @@ func main() {
 	}
 
 	if err := app.Run(os.Args); err != nil {
-		log.Error().Err(err).Msg("run orbit failed")
+		logging.LogErrIfEnvNotSet(constant.SilenceEnrollLogErrorEnvVar, err, "run orbit failed")
```

Comment:
Why do we need this change?

Reply:
Thought I saw that error in the issue, but I'm not seeing it during testing.  I'll revert this.

Reply:
OK, this is logged every time Orbit exits with an error.